deploy/deployment.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: trace-context-injector-webhook-deployment
+  labels:
+    app: trace-context-injector
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: trace-context-injector
+    spec:
+      containers:
+        - name: trace-context-injector
+          image: staging-k8s.gcr.io/trace-context-injector:latest
```

Comment:
I think this should just be `k8s.gcr.io`.  We push to staging, which is graduated to `k8s.gcr.io`

Reply:
done